src/main/java/org/commonjava/util/partyline/lock/UnlockStatus.java

```diff
@@ -2,9 +2,9 @@
 
 public class UnlockStatus
 {
-    boolean unlocked;
-    boolean dominanceChanged;
-    LockLevel dominantLockLevel;
+    public boolean unlocked;
```

Comment:
Either make getter methods for these, or set these fields `final` and set them up using only the constructor.